transforms/helpers/util/index.ts

```diff
@@ -172,6 +172,78 @@ export const LIFECYCLE_HOOKS = new Set([
   'drop',
 ]);
 
+// DANGER! This assumes that the correct decorator is imported just because of
```

Comment:
Can we use telemetry to double check?

Reply:
Selfishly, we might want to keep this assumption because https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse-common/addon/utils/decorators.js

😬

Reply:
(Also, I'm not sure we'd need the telemetry to tell bc we can check the import statements.)